front/app/utils/configs/participationMethodConfig/index.tsx

```diff
@@ -276,6 +278,7 @@ const votingConfig: ParticipationMethodConfig = {
   showInputCount: false,
   formEditor: 'simpleFormEditor',
   showIdeaFilters: false,
+  inputsPageSize: 100,
```

Comment:
Added this to the votingConfig, but since the default value is already 24 I didn't add this to the ideationConfig. Let me know if you think I should though for clarity :)

Reply:
Let's also add `inputsPageSize: 24` to the ideation config? Functionally the same, but more explicit